### DISCUSSION TITLE
Welcome to happy-birthday Discussions!

### DISCUSSION THREAD

**Discussion by faahim** [faahim/happy-birthday#16] [General]
## 👋 Welcome!

This is a place to discuss anything and everything about this project. Problem setting it up, ideas, anything!

**Comment by Samsonroyal** [comment]
Hello here :)

**Reply by faahim** [reply]
Hey @Samsonroyal 

How are you doing today?

Thanks for checking out the project! Lemme know if you have any questions or anything! 🙂

**Reply by Samsonroyal** [reply]
Hello @faahim I've checked it out and i can say it's awesome. One of my friend's birthdays is coming up in a few days so I wanted to do something nerdy for him. I'll surely let you know when anything/issue or problem comes up

**Reply by faahim** [reply]
Sounds amazing! Happy to help in whatever way possible!

Also, I've done this quite a long time ago and was secretly planning to bring some upgrades to it. So if you have any idea that the project currently doesn't support, please feel free to drop them here! :)

**Reply by Samsonroyal** [reply]
Oh yes I’ve noticed something; the hat icon; is there a way we can reposition it to sit over the head of the subject in an image perfectly? And the other issue is about resizing the images. I guess they’re related in a way.

**Comment by heiooo** [comment]
Hello, I'd like to ask you why the code of the birthday letter page in github can't be displayed when you open the html after downloading it to your computer

**Reply by SYSU2024SX** [reply]
me, too! I do not why

**Reply by faahim** [reply]
Hey 👋 

This is because the project requires some build steps before it can run in the browser locally. 

Once you've cloned/downloaded the project, open it in a terminal and run `npm install && npm run start`. That should fix the issue. Please note that you'll need to have Node.js installed on your computer.

**Comment by SYSU2024SX** [comment]
Thank you for helping mm!
I have just discovered that the computer refused to access local files with the 'fetch' function in order to protect file security. I have now resolved this issue. Thank you very much for answering this question！

---Original---
From: "Afiur Rahman ***@***.***&gt;
Date: Tue, Jun 4, 2024 21:45 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [faahim/happy-birthday] Welcome to happy-birthday Discussions!(Discussion #16)




 
Hey 👋
 
This is because the project requires some build steps before it can run in the browser locally.
 
Once you've cloned/downloaded the project, open it in a terminal and run npm install &amp;&amp; npm run start. That should fix the issue. Please note that you'll need to have Node.js installed on your computer.

**Comment by gokullaxman** [comment]
for me the animations are working

**Comment by wecoded-dev** [comment]
ewishWall is really, really amazing! 😍 I liked it so, so much and found it absolutely excellent. I'm super excited for the new templates—please add more! 🙏 I'm very excited about this.

If you'd like, I can promote your website a lot. I can do a ton of advertising for it, but you won't have to pay me; I'll do it for free. 👍
